packages/mediapipe-core/lib/universal_mediapipe_core.dart

```diff
@@ -122,3 +172,18 @@ class ClassifierOptions extends BaseClassifierOptions {
   @override
   double? get scoreThreshold => throw UnimplementedError();
 }
+
+/// {@macro EmbedderOptions}
+class EmbedderOptions extends BaseEmbedderOptions {
```

Comment:
Why do we define this class twice? Can it just be shared from the set of core classes with the task-specific ones?